### ISSUE TITLE
MultiBit my bitcoin isnt sending at all

### ISSUE THREAD

**Issue by u7alll** [blockchain/blockchain-wallet-v4-frontend#950]
So, 3 weeks ago i sent all the bitcoin on my multibit account to blockchain, yet the transaction has still not gone through and is still stuck on a triangle (indicating it has been seen by 0 peers). How can I get my bitcoin back? I know for a fact that i haven't put in an incorrect bitcoin address so that is not the issue.

I have read elsewhere on the internet that I can get my bitcoin back if I export my private keys yet I am getting very confused on how this works..

All the best, Matt

**Comment by plondon** [comment]
Hi Matt,

The only way to know if your bitcoin was sent to an address generated by the blockchain wallet is to see the transaction on the blockchain. Did multibit send you a transaction? It looks like this: https://www.blockchain.com/btc/tx/4b07f904f56e5af4906e24aab210ba29384b874b02d27f0d33385e398e6a8582

**Comment by u7alll** [comment]
Yes but when i click view info on blockchain it says transaction not found

**Comment by plondon** [comment]
What is the transaction id/hash?

**Comment by plondon** [comment]
Since this is most likely not an issue with the wallet code, you may want to file a ticket with multibit support team if you have not already, or our own support team support.blockchain.com

**Comment by u7alll** [comment]
Yeah, sadly multibit have noone working for them anymore

**Comment by plondon** [comment]
Sorry to hear that :(.

Can't stress enough how important it is to own your private keys.

Closing because this is not a wallet issue

**Comment by u7alll** [comment]
I do own my private keys

**Comment by plondon** [comment]
Does multibit give you a recovery phrase?

**Comment by u7alll** [comment]
No i think not, but it allows me to export my wallets

**Comment by plondon** [comment]
Without giving me the contents for that can you share the structure of it? What does it look like?

**Comment by plondon** [comment]
If it's 12 words for example you can use that to restore your wallet in blockchain here https://login.blockchain.com/#/recover

**Comment by u7alll** [comment]
Im afraid multibit never gave me a 12 word passphrase

**Comment by u7alll** [comment]
should i screenshot some photos of my multibit wallet?

**Comment by u7alll** [comment]
But the bitcoin is stuck on multibit not blockain

**Comment by plondon** [comment]
> No i think not, but it allows me to export my wallets

What does this data consist of

**Comment by u7alll** [comment]
it consists of giving me the wallet adress that i cam import, yet when i
tried importing one of my wallets it didnt give me the funds :/

**Comment by plondon** [comment]
Is it a public key or private key?

Do any of them show a balance here: https://www.blockchain.com/btc/address/YOUR_ADDRESS_HERE

**Comment by u7alll** [comment]
it says both wallets have 0 bitcoin but i can still see that 0.3 bitcoin
has been transferred out one of the wallets and has not been seen by any
peers
